logos/logos-ui/src/app/shared/components/icon-tile/icon-tile.scss

```
@@ -7,20 +7,9 @@
   font-weight: var(--font-weight-bold);
   user-select: none;
   line-height: var(--line-height-none);
-  box-shadow: 0 2px 4px rgb(var(--shadow-color) / var(--opacity-soft));
 
-  .pi { font-size: inherit; line-height: 1; }
+  background: rgb(var(--color-primary-500) / var(--opacity-soft));
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)